config/database.yml

```diff
@@ -6,7 +6,11 @@ default: &default
 
 development:
   <<: *default
-  database: dss_dev
+  host: <%= ENV["lando_dss_database_conn_host"] %>
+  port: <%= ENV["lando_dss_database_conn_port"] %>
+  username: <%= ENV["lando_dss_database_creds_user"] %>
+  password: <%= ENV["lando_dss_database_creds_password"] %>
+  database: dss_development
 
```

Comment:
It would be nice to use these lando environment variables for the test database as well, to allow running tests locally.